src_python/fl_client_shed.py

```diff
@@ -0,0 +1,237 @@
+import sys
+import logging
+import gc
+
+arg_names = [
+    'client_id',
+    'path_weights',
+    'path_nodes',
+    'path_edges',
+    'graph_id',
+    'partition_ids',
+    'epochs',
+    'IP',
+    'PORT'
+]
+
+args = dict(zip(arg_names, sys.argv[1:]))
+
+client_id = args['client_id']
+
+logging.basicConfig(
+    level=logging.INFO, 
+    format='%(asctime)s : [%(levelname)s]  %(message)s',
+    handlers=[
+        logging.FileHandler(f'client_shed_{client_id}.log'),
+        logging.StreamHandler(sys.stdout)
+    ]
+)
+
+import socket
```

Comment:
Lets move these imports also to the top of the file

Reply:
When I do this python logging does not work as expected

Reply:
I see